cmd/build

```diff
@@ -1,51 +1,48 @@
 #!/bin/bash -e
+# The following are expected modes of builds
+# cmd/build <- Builds all docker images
```

Comment:
hmmm, I would make sure these are coherent.... right now, you have:
push == push
pull == pull then build
_nothing_ == build all

I would make it more explicit, so:
pull == pull
build == build all
_nothing_ == pull then build

Reply:
I'll change the behavior to follow this.